.github/workflows/ci.yml

```diff
@@ -31,8 +31,6 @@ jobs:
         run: make test-swift
       - name: Build platforms ${{ matrix.config }}
         run: CONFIG=${{ matrix.config }} make build-all-platforms
-      # - name: Build for library evolution
-      #   run: make build-for-library-evolution
```

Comment:
if we wanna support this we need to go all the way back to Issue Reporting to fix some things. But also I'm not sure if we really need to support this unless someone is depending on it (which clearly no one is since it's been broken for awhile).